packages/web-experience/src/web-experience-processor.ts

```diff
@@ -125,19 +126,36 @@ export function createItemFromTemplate(
           id: exbModel.item.id,
           authentication: destinationAuthentication
         }).then(() => {
-          return Promise.resolve({
-            id: "",
-            type: template.type,
-            postProcess: false
-          });
+          return Promise.resolve(_generateEmptyCreationResponse(template.type));
         });
       } else {
         // finally, return ICreateItemFromTemplateResponse
         return {
+          item: {
+            ...template,
+            ...exbModel
+          },
           id: exbModel.item.id,
           type: template.type,
           postProcess: false
         };
       }
     });
 }
+
+// ------------------------------------------------------------------------------------------------------------------ //
+
+/**
+ * Flags a failure to create an item from a template.
+ * @return Empty creation response
+ */
+export function _generateEmptyCreationResponse(
```

Comment:
@MikeTschudi why all the redundancy with this function across so many modules?

Reply:
You're right--I'll merge them into `common`. Thanks for pointing it out.